README.md

```diff
@@ -52,7 +52,7 @@ module "example" {
 
```

Comment:
I was going to mention updating the `Notes` section for the removal of the `dns` server when I noticed that neither `dns` nor the `security_groups` folders are mentioned. Would you update it to reflect the `security_groups` directory?

Reply:
Are you referring to the need to run `terraform init` on subdirectories?  This is not necessary for _in situ_ submodules; you can run `pre-commit run -a` locally to verify this.

Reply:
Ahh right, because it gets treated as part of the configuration in the root directory through usage. Apologies.